web/cobrands/northamptonshire/assets.js

```diff
@@ -56,6 +56,7 @@ var layers = [
   ],
   "layer_name": "Tree",
   "styleid": '5d481376fe2ad8059ce44ef2',
+  "disable_pin_snapping": "true",
```

Comment:
As discussed, and as per your comment above, this isn't actually to do with the existing `disable_pin_snapping` code which is for disabling a different type of pin snapping. So whilst you could have a different name - `disable_nearest_asset_snapping` perhaps - I think best solution is actually to set `snap_threshold` to 0 directly on this object, as the existing code checks that.